sbin/build.sh

```diff
@@ -397,7 +397,7 @@ buildSharedLibs() {
     local gradleJavaHome=$(getGradleHome)
     echo "Running gradle with $gradleJavaHome"
 
-    JAVA_HOME="$gradleJavaHome" GRADLE_USER_HOME=./gradle-cache ./gradlew --no-daemon clean uberjar
+    JAVA_HOME="$gradleJavaHome" GRADLE_USER_HOME=./gradle-cache bash ./gradlew --no-daemon clean uberjar
```

Comment:
Is bash on all platforms?

Reply:
Much as I'd rather they weren't, our build scripts are full of fairly sophisticated bash-isms, so it has to be on all our systems already.

Reply:
Ack